packages/@webex/contact-center/src/types.ts

```diff
@@ -0,0 +1,108 @@
+export enum HTTP_METHODS {
+  GET = 'GET',
+  POST = 'POST',
+  PATCH = 'PATCH',
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+}
+
+export type WebexRequestPayload = {
+  method?: HTTP_METHODS;
+  uri?: string;
+  addAuthHeader?: boolean;
+  headers?: {
+    [key: string]: string | null;
+  };
+  body?: object;
+  statusCode?: number;
+  json?: boolean;
+};
+
+type Listener = (e: string, data?: unknown) => void;
+type ListenerOff = (e: string) => void;
+
+export type ServiceHost = {
+  host: string;
+  ttl: number;
+  priority: number;
+  id: string;
+  homeCluster?: boolean;
+};
+
+export interface CCConfig {
```

Comment:
can we have tsdoc style descriptions?

Reply:
This is not going to be type publicly exposed. Do we still need to add tsdoc descriptions